template/page-events.html.jinja

```diff
@@ -371,6 +371,24 @@
     "image": "AMAs.png",
     "end_date": "June 24, 2023"
     },
+    { "name": "XRPL BUIDLERS BOOTCAMP",      
```

Comment:
I was wondering if this was intentionally stylized as "BUIDL" or just a typo, given that the URL has "builders" spelled correctly, but the link makes it pretty clear it's by design (see screenshot below).

So, :+1: no changes needed here, just FYI for any other reviewers.


![2023-10-09_162021_806221024](https://github.com/XRPLF/xrpl-dev-portal/assets/7515597/916a0766-2ac1-4656-9eef-92bc80c9509a)

Reply:
I understand this is an intentional typo, but I can't help but wonder how this is expected to be pronounced aloud. Just like "builders" ...?